weave-js/src/components/PagePanelComponents/Home/Browse3/pages/wfReactInterface/interface.ts

```diff
@@ -35,6 +35,17 @@ import {Call as CallTreeSpan} from '../../../Browse2/callTree';
 import {useRuns} from '../../../Browse2/callTreeHooks';
 import {PROJECT_CALL_STREAM_NAME, WANDB_ARTIFACT_REF_PREFIX} from './constants';
 
+const OP_CATEGORIES = [
```

Comment:
I think these should be e.g.:
`const OP_CATEGORIES = ['train', ...] as const`

Reply:
👍